src/Schema/Events/Search.ts

```diff
@@ -88,6 +88,86 @@ export interface SearchedWithNoResults {
   query: string
 }
 
+/**
+ * A user selects search pill
+ *
+ * This schema describes events sent to Segment from [[selectedPillFromSearch]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "selectedPillFromSearch",
+ *    context_module: "header",
+ *    context_owner_type: "home",
+ *    destination_owner_type: "search",
+ *    query: "andhuwe"
+ *    pill: "artwork"
+ *  }
+ * ```
+ */
+export interface SelectedPillFromSearch {
+  action: ActionType.selectedPillFromSearch
+  context_module: ContextModule
+  context_owner_type: PageOwnerType
+  context_owner_id?: string
+  context_owner_slug?: string
+  destination_owner_type?: PageOwnerType
+  query: string
+  pill: string
```

Comment:
**question(non-blocking):** Should we maybe define a type for pill names? 🤔

Reply:
I was thinking since we have specific pill names it could be something like:
```ts
pill: Pill

type Pill = "Artist" | "Artwork" 
``` 

etc, but I checked eigen and it doesn't follow the same so 🤷‍♂️ feel free to ignore

Reply:
I agree that we should type the allowed values here in cohesion. I think we can probably use the `key` values [here](https://github.com/artsy/eigen/blob/main/src/app/Scenes/Search/constants.ts#L25-L83) as an initial set.

So something like:

```typescript
type PillTypes = "top" | "artwork" | "artist" | "article" | "sale" | "artist_series" | "marketing_collection" | "fair" | "partner_show" | "PartnerGallery"
```

A little strange with the different casing between `partner_show` and `PartnerGallery` but might be fine as long as it's well-documented in this way.